src/shell/zsh.rs

```diff
@@ -51,4 +53,8 @@ impl Shell for Zsh {
             autoload_hook = autoload_hook
         ))
     }
+
+    fn format_path(&self, path: &str) -> Result<String> {
+        to_unix_path(path).context("Failed to convert Windows path to Unix")
```

Comment:
I don't think we need to add that context, as it's implied from the function itself